.eslintrc

```diff
@@ -85,6 +85,42 @@
       }
     ],
     "import/no-cycle": "error",
+    "import/order": [
+      "error",
+      {
+        "groups": [
+          ["external", "builtin"],
```

Comment:
Why have `builtin`'s after `external`? 🤔 It's the default

I too like to see `"react"` on top, but if there's a NodeJS import in a react file, probably something's wrong - and it being on top helps identifying it

Reply:
Just a personal preference. I'll change this.